.github/ISSUE_TEMPLATE/01-bug.yml

```
@@ -0,0 +1,90 @@
+name: General Bug
+labels: ["status/needs-triage", "type/bug"]
+description: Report a general problem with Bottlerocket
+body:
+  - type: markdown
+    attributes:
+      value: |
+        This bug report form is for **general issues** with Bottlerocket. 
+        
+        A few tips before submitting a new issue:
+        - Search for similar issues, including [closed issues](https://github.com/bottlerocket-os/bottlerocket/issues?q=is%3Aissue+is%3Aclosed),
+        - Include details about the environment you're running in,
+        - Include any error messages you received, with any required context.
+  - type: dropdown
+    id: variant
+    attributes:
+      label: Variant
+      options:
+        - aws-k8s-1.21
+        - aws-k8s-1.22
+        - aws-k8s-1.23
+        - aws-k8s-1.24
+        - aws-ecs-1
+        - vmware-k8s-1.21
+        - vmware-k8s-1.22
+        - vmware-k8s-1.23
+        - vmware-k8s-1.24
+        - metal-k8s-1.21
+        - metal-k8s-1.22
+        - metal-k8s-1.23
+        - metal-k8s-1.24
+        - Other variant (specify in 'I expected to happen')
+    validations:
+      required: true
+  - type: dropdown
+    id: version
+    attributes:
+      label: Bottlerocket version
+      options:
+        - 1.11.x
+        - 1.10.x
+        - 1.9.x
+        - 1.8.x
+        - 1.7.x
+        - 1.6.x
+        - 1.5.x
+        - 1.4.x
+        - 1.3.x
+        - 1.2.x
+        - 1.1.x
+        - 1.0.x
```

For usability reasons, and to discourage the idea that very old versions are still supported, it might be useful to cap this at the three or four most recent releases:
```suggestion
        - 1.11.x
        - 1.10.x
        - 1.9.x
        - Other version (specify in 'I expected to happen')
```